From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Tue, 1 Feb 2022 15:33:27 +0200
Subject: [PATCH] Disable building docs

---
 Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index f347162269cb1fe38ea81f0c6c170bbe67452b00..8edab4c1863d906c9b5910f948d5622c6e144e3a 100644
--- a/Makefile
+++ b/Makefile
@@ -110,10 +110,6 @@ SHARED_LIBS = $(LIBGIFSO) $(LIBUTILSO)
 STATIC_LIBS = libgif.a libutil.a
 
 all: shared-lib static-lib $(UTILS)
-ifeq ($(UNAME), Darwin)
-else
-	$(MAKE) -C doc
-endif
 
 $(UTILS):: $(STATIC_LIBS)
 
@@ -169,7 +165,7 @@ spellcheck:
 ifeq ($(UNAME), Darwin)
 install: all install-bin install-include install-lib
 else
-install: all install-bin install-include install-lib install-man install-doc
+install: all install-bin install-include install-lib
 endif
 
 install-bin: $(INSTALLABLE)
