From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Tue, 1 Feb 2022 15:33:27 +0200
Subject: [PATCH] Disable building docs

---
 Makefile | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/Makefile b/Makefile
index 87966a96cd4f7325da09b64348f18079f5a22a25..6842522073f2c8d280d96024d96c3a99070f7580 100644
--- a/Makefile
+++ b/Makefile
@@ -88,10 +88,6 @@ LIBUTILSOMAJOR	= libutil.$(LIBMAJOR).$(SOEXTENSION)
 endif
 
 all: $(LIBGIFSO) libgif.a $(LIBUTILSO) libutil.a $(UTILS)
-ifeq ($(UNAME), Darwin)
-else
-	$(MAKE) -C doc
-endif
 
 $(UTILS):: libgif.a libutil.a
 
@@ -132,7 +128,7 @@ reflow:
 ifeq ($(UNAME), Darwin)
 install: all install-bin install-include install-lib
 else
-install: all install-bin install-include install-lib install-man
+install: all install-bin install-include install-lib
 endif
 
 install-bin: $(INSTALLABLE)
